test

```diff
@@ -96,22 +96,40 @@ function cov_pass {
 
 	mkdir -p "$COVERDIR"
 
+	# convert COVERDIR to absolute path
+	# e2e needs absolute path to put its coverfiles
```

Comment:
it appears to me that the e2e tests has its path relative to the e2e pkg. 
https://github.com/coreos/etcd/blob/master/e2e/main_test.go#L23-L24

if that's the case when `pwd=coreos/etcd` and ` COVERDIR=../coverage`, the e2e will then put cover files to `coreos/etcd/coverage` instead of the intended place `coreos/coverage`.